librosa/core/convert.py

```diff
@@ -1915,7 +1943,10 @@ def D_weighting(
         )
     )
 
-    return weights if min_db is None else np.maximum(min_db, weights)
+    if min_db is None:
+        return weights
+    else:
+        return np.maximum(min_db, weights)
```

Comment:
Are changes like this just because of personal preference? From a type checking perspective they aren't necessary, right?

Reply:
They aren't necessary, but they're A) more readable, and B) made it easier to trace errors down to specific parts.  The --pretty option only goes so far with these things.